.github/actions/duvet/action.yml

```diff
@@ -69,15 +69,21 @@ runs:
           REPORT_PATH=$(dirname "${{ inputs.report-script }}")/report.html
         fi
 
-        TARGET="${{ github.sha }}/compliance.html"
-        aws s3 cp "$REPORT_PATH" "s3://${{ inputs.aws-s3-bucket-name }}/$TARGET" --acl private --follow-symlinks
+        TARGET_SHA="${{ github.sha }}/compliance.html"
+        aws s3 cp "$REPORT_PATH" "s3://${{ inputs.aws-s3-bucket-name }}/$TARGET_SHA" --acl private --follow-symlinks
+
+        # Only upload to latest if the event is push to main
+        if [ "${{ github.event_name }}" == "push" ]; then
+          TARGET_LATEST="latest/compliance.html"
+          aws s3 cp "$REPORT_PATH" "s3://${{ inputs.aws-s3-bucket-name }}/$TARGET_LATEST" --acl private --follow-symlinks
```

Comment:
Would it be possible to copy from the `$TARGET_SHA` folder to the `$TARGET_LATEST` rather than re-uploading to s3?

Something like:
`aws s3 cp s3://${{ inputs.aws-s3-bucket-name }}/$TARGET_SHA/ s3://${{ inputs.aws-s3-bucket-name }}/$TARGET_LATEST/ --recursive`

Reply:
`--recursive` might only be needed for folders, some of these uploads are single files